### PR TITLE
Update contributing guidelines

### PR DESCRIPTION
**Pull request by sternenseemann**
Any comments?

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -52,4 +52,21 @@ Please talk to the people! If everything is coordinated well, development will b
 These are the two communication channels we use:
 
 * Github issues for in depth discussion of concrete problems
-* IRC (`#jugendhackt` on freenode) for pretty much everything else (ranging from technical to more general to off-topic discussion)
+* IRC (`#paketmagie` on freenode) for pretty much everything else (ranging from technical to more general to off-topic discussion)
```
Comment (froozen) [inline CONTRIBUTING.adoc]: Wouldn't `#ricochet` or something along those lines be better?

**Comment by froozen** [comment]
The issues section should also explain how to close issues in your commits and that people should assign themselves to issues they want to solve.

**Comment by sternenseemann** [comment]
Any further thoughts?

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -52,4 +52,23 @@ Please talk to the people! If everything is coordinated well, development will b
...
+
+=== Issues
+
+* Formulate clear issue titles
+* Write clear and concise comments
+* Label issues if you are allowed to
+* For bugs / errors supply…
+** … the information how to reproduce
+** … the error message
+* If you plan to resolve the issue, asign yourself to it.
```
Comment (photm5) [inline CONTRIBUTING.adoc]: Should be “assign.”

**Comment by sternenseemann** [comment]
_Bump_

**Comment by froozen** [comment]
I don't have anything to add.

**Comment by photm5** [comment]
I’m ok with it.

**Pull request merged**